components/alibi-detect-server/adserver/cd_model.py

```diff
@@ -5,6 +5,55 @@
 from .numpy_encoder import NumpyEncoder
 from alibi_detect.utils.saving import load_detector, Data
 from adserver.base import AlibiDetectModel
+from seldon_core.user_model import SeldonResponse
+
+
+def _drift_to_metrics(drift):
+    metrics = []
+
+    batch_score = drift.get("batch_score")
+    if batch_score:
+        metrics.append(
+            {
+                "key": "seldon_metric_drift_batch_score",
+                "value": batch_score,
+                "type": "GAUGE",
+            }
+        )
+
+    feature_score = drift.get("feature_score")
+    if feature_score:
+        metrics.append(
+            {
+                "key": "seldon_metric_drift_feature_score",
+                "value": feature_score,
+                "type": "GAUGE",
```

Comment:
Why is this a Gauge?

Reply:
The thought process is that feature_score is a metric related to a feature that changes on each request, as opposed to a total aggregate, would you say that COUNT would be more appropriate?